packages/react/package.json

```
@@ -85,7 +85,7 @@
   },
   "dependencies": {
     "@github/combobox-nav": "^2.1.5",
-    "@github/markdown-toolbar-element": "^2.1.0",
+    "@github/markdown-toolbar-element": "2.2.3",
```

Do we want to copy the pattern of other deps here?

```suggestion
    "@github/markdown-toolbar-element": "^2.2.3",
```